code/ship/ship.cpp

```diff
@@ -11782,6 +11759,30 @@ int ship_fire_primary(object * obj, int force, bool rollback_shot)
 
```

Comment:
Since we now do autoaim per weapon, we should move the variable declaration of ``float autoaim_fov`` from the top of the function to the top of this for loop, to make sure that it doesn't erroneously use it later when it must be defined per-weapon